proto/xudrpc.proto

```diff
@@ -422,6 +422,8 @@ message CreateNodeResponse {
   repeated string seed_mnemonic = 1;
   // The list of lnd clients that were initialized.
   repeated string initialized_lnds = 2;
+  // The list of connext wallets that were initialized.
+  repeated string initialized_connexts = 3;
```

Comment:
Think we can change this to a boolean `initialized_connext` since we're not actually using the values in this array for the CLI.